pkg/container/firecracker.go

```diff
@@ -83,6 +85,22 @@ func ExecuteFirecracker(vm *api.VM, dhcpIfaces []DHCPInterface) error {
 		MetricsFifo: metricsSocketPath,
 	}
 
+	// Add the volumeMount mappings to the VM
+	for i, volumeMount := range vm.Spec.Storage.VolumeMounts {
+		volumePath := path.Join(constants.IGNITE_SPAWN_VOLUME_DIR, volumeMount.Name)
+		if !util.FileExists(volumePath) {
+			log.Warnf("Skipping volume mount %q: nonexistent volume", volumeMount.Name)
+			continue // Skip all volume mounts with nonexistent paths
+		}
+
+		cfg.Drives = append(cfg.Drives, models.Drive{
+			DriveID:      firecracker.String(strconv.Itoa(i + 2)),
+			IsReadOnly:   firecracker.Bool(false),
```

Comment:
Add a TODO to support readonly volumes eventually?

Reply:
Added in ca75db1.